### DISCUSSION TITLE
🉐️ What languages should we add to Upscayl?

### DISCUSSION THREAD

**Discussion by NayamAmarshe** [upscayl/upscayl#899] [Announcements]
We'll be adding more languages to Upscayl Desktop in the next version.

Let me know the language that you'd like to see 👇

(The ones with most votes will win 😉)

**Comment by albanobattistella** [comment]
> We'll be adding more languages to Upscayl Desktop in the next version.
> 
> Let me know the language that you'd like to see 👇
> 
> (The ones with most votes will win 😉)

Hi
Italian.
thank you.
A.

**Reply by Handsome-manss** [reply]
Chinese

**Comment by aaronliu0130** [comment]
I can add Chinese myself if we manage to set up translations

**Comment by presi300** [comment]
I can help with Bulgarian

**Comment by lukazevendetta** [comment]
i can do for portuguese

**Comment by Handsome-manss** [comment]
l need Chinese

**Comment by ghost** [comment]
Russian
